### PR TITLE
Link to workflow status badge article

### PR DESCRIPTION
**Pull request by Simran-B**
### Why:

[Status badges for workflow runs](https://docs.github.com/en/actions/guides/about-continuous-integration#status-badges-for-workflow-runs) links to "Configuring a workflow", which redirects to "Learn GitHub Actions", but it's just an overview page and there is no article about workflow status badges.

### What's being changed:

- Link to the actual article about workflow status badges
- Update the link "Setting up continuous integration using GitHub Actions" under Further reading to where it redirects you and update the link label as well
- Make `README.md` in reusable text an inline code element
- Rename the reusable from `actions-workflow-status-badge-into.md` to `…-intro.md` - looked like a typo to me

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### CODE REVIEW

File: content/actions/guides/setting-up-continuous-integration-using-workflow-templates.md [hunk 1]
```diff
@@ -38,14 +38,15 @@ After you set up CI, you can customize the workflow to meet your needs.
 
 Once a push is made to your repository, you can follow the status and detailed logs of your continuous integration workflow run on {% data variables.product.prodname_dotcom %} and receive customized notifications. For more information, see "[Configuring notifications](/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#github-actions-notification-options)" and "[Managing a workflow run](/articles/managing-a-workflow-run)."
 
-{% data reusables.repositories.actions-workflow-status-badge-into %}
+{% data reusables.repositories.actions-workflow-status-badge-intro %}
 
 For more information, see "[Learn {% data variables.product.prodname_actions %}](/actions/learn-github-actions)."
```
Comment (Simran-B) [inline content/actions/guides/setting-up-continuous-integration-using-workflow-templates.md:43]:
```suggestion
For more information, see "[Adding a workflow status badge](/actions/managing-workflow-runs/adding-a-workflow-status-badge)."
```

File: content/actions/guides/setting-up-continuous-integration-using-workflow-templates.md [hunk 2]
```diff
@@ -38,14 +38,15 @@ After you set up CI, you can customize the workflow to meet your needs.
...
 
 ## Further reading
 
 - "[About continuous integration](/articles/about-continuous-integration)"
 - "[Managing a workflow run](/articles/managing-a-workflow-run)"
+- "[Adding a workflow status badge](/actions/managing-workflow-runs/adding-a-workflow-status-badge)"
```
Comment (Simran-B) [inline content/actions/guides/setting-up-continuous-integration-using-workflow-templates.md:49]:
```suggestion
- "[Learn {% data variables.product.prodname_actions %}](/actions/learn-github-actions)"
```

**Review by Simran-B — COMMENTED** [review]
I think the links for workflow badge status and learning GitHub Actions should actually be swapped

**Comment by janiceilene** [comment]
Thanks so much @Simran-B! I'll get this triaged for review 💝

File: content/actions/guides/about-continuous-integration.md [hunk 3]
```diff
@@ -82,13 +82,13 @@ You won't be able to merge the pull request if your repository is configured to
...
-{% data reusables.repositories.actions-workflow-status-badge-into %}
+{% data reusables.repositories.actions-workflow-status-badge-intro %}
 
-For more information, see "[Configuring a workflow](/articles/configuring-a-workflow)."
+For more information, see "[Adding a workflow status badge](/actions/managing-workflow-runs/adding-a-workflow-status-badge)."
 
 ## Further reading
 
-- "[Setting up continuous integration using {% data variables.product.prodname_actions %}](/articles/setting-up-continuous-integration-using-github-actions)"
+- "[Setting up continuous integration using workflow templates](/actions/guides/setting-up-continuous-integration-using-workflow-templates)"
```
Comment (lecoursen) [inline content/actions/guides/about-continuous-integration.md:91]: We actually don't include articles in Further reading that are included in the article's sidebar, as this one is, so we can remove this entirely ([ref](https://github.com/github/docs/blob/main/contributing/content-model.md#further-reading)).

```suggestion
```

**Review by lecoursen — DISMISSED** [review]
This looks great! I left one comment, which I'll apply now, and then merge!

**Review by lecoursen — DISMISSED** [review]

**Review by lecoursen — APPROVED** [review]

**Pull request merged**